src/app/Startup.cs

```diff
@@ -36,6 +36,7 @@ public void ConfigureServices(IServiceCollection services)
             // set json serialization defaults
             services.AddControllers().AddJsonOptions(options =>
             {
+                options.JsonSerializerOptions.WriteIndented = true;
```

Comment:
I don't think you can do this because all of the json responses will be indented which dramatically increases the payload size.

Reply:
it actually doesn't even work now that we're writing with StringBuilder (it used to when I passed an object back). removed.

Reply:
...removed